vk_mem_alloc.h

```diff
@@ -16287,7 +16287,7 @@ VMA_CALL_PRE VkResult VMA_CALL_POST vmaCreateAllocator(
 {
     VMA_ASSERT(pCreateInfo && pAllocator);
     VMA_ASSERT(pCreateInfo->vulkanApiVersion == 0 ||
-        (VK_VERSION_MAJOR(pCreateInfo->vulkanApiVersion) == 1 && VK_VERSION_MINOR(pCreateInfo->vulkanApiVersion) <= 2));
+        (VK_VERSION_MAJOR(pCreateInfo->vulkanApiVersion) == 1 && VK_VERSION_MINOR(pCreateInfo->vulkanApiVersion) <= 3));
```

Comment:
it would be best to update the whole vk_mem_alloc.h from official one provided in the vulkan sdk

Reply:
Yes. I just wanted to have a quick Vulkan Compute hello world to test, yours was the first at Google. :)
(Meaning: I dont' care too much.)